In [3]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
df_games = pd.read_parquet('data_games.parquet')

Análisis Exploratorio de Datos (EDA)

-Estadistica descriptiva

In [3]:
df_games.describe()

price
count  29930.000000
mean       8.504181
std       15.922590
min        0.000000
25%        1.990000
50%        4.990000
75%        9.990000
max      995.000000

observamos que el valor medio en Price es de 8.5 y su maximo de 995

In [4]:
df_games.info()

<class 'pandas.core.frame.DataFrame'>
Index: 29930 entries, 0 to 32131
Data columns (total 9 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   publisher     23989 non-null  object 
 1   genres        29930 non-null  object 
 2   title         29930 non-null  object 
 3   release_date  29930 non-null  object 
 4   specs         29268 non-null  object 
 5   price         29930 non-null  float64
 6   early_access  29930 non-null  bool   
 7   id            29930 non-null  object 
 8   developer     28707 non-null  object 
dtypes: bool(1), float64(1), object(7)
memory usage: 2.1+ MB


De 29930 filas, las columnas publisher, specs y developer poseen valores nulos

In [5]:
# Para el gráfico de barras de la variable "genres"
plt.figure(figsize=(12, 6))
df_games['genres'].value_counts().head(10).plot(kind='bar')
plt.title('Top 10 Géneros de Juegos')
plt.xlabel('Género')
plt.ylabel('Frecuencia')
plt.xticks(rotation=45)
plt.show()